monai/transforms/utility/array.py

```diff
@@ -1715,3 +1715,101 @@ def __call__(self, img: NdarrayOrTensor, meta_dict: Mapping | None = None) -> Nd
         if self._do_transform:
             img = self.filter(img)
         return img
+
+
+class CoordinateTransform(InvertibleTransform, Transform):
```

Comment:
Please add the type hints for the member function methods. Thanks!

Reply:
Added, thanks.